pkg/services/app/remove.go

```diff
@@ -74,7 +75,12 @@ func (a *App) Remove(params RemoveParams) error {
 		return fmt.Errorf("failed to obtain config URL and branch: %w", err)
 	}
 
-	remover, err := a.cloneRepo(a.ConfigGit, cloneURL, branch, params.DryRun)
+	normalizedUrl, err := gitproviders.NewNormalizedRepoURL(cloneURL)
```

Comment:
Having this so close to the `cloneRepo` method where it is consumed makes me wonder if we can get a `NormalizedRepoURL` somewhere earlier in the stack. Just a thought.

Reply:
I looked and we are not using url at any point before this.